app/Http/Controllers/WikiEntityImportController.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\WikiEntityImportStatus;
+use Illuminate\Http\Request;
+use App\Wiki;
+use App\WikiEntityImport;
+use App\Jobs\WikiEntityImportDummyJob;
+use Carbon\Carbon;
+
+class WikiEntityImportController extends Controller
+{
+    public function get(Request $request): \Illuminate\Http\JsonResponse
+    {
+        $wiki = Wiki::find($request->input('wiki'));
+        if (!$wiki) {
+            abort(404, 'No such wiki');
+        }
+
+        $imports = $wiki->wikiEntityImports()->get();
+        return response()->json(['data' => $imports]);
+    }
+
+    public function create(Request $request): \Illuminate\Http\JsonResponse
+    {
+        $wiki = Wiki::find($request->input('wiki'));
+        if (!$wiki) {
+            abort(404, 'No such wiki');
+        }
+
+        $imports = $wiki->wikiEntityImports()->get();
+        foreach ($imports as $import) {
+            if ($import->status === WikiEntityImportStatus::Success) {
+                return response()
+                    ->json(['error' => 'Wiki "'.$wiki->domain.'" already has performed a successful entity import'])
+                    ->setStatusCode(400);
+            }
+            if ($import->status === WikiEntityImportStatus::Pending) {
+                return response()
+                    ->json(['error' => 'Wiki "'.$wiki->domain.'" currently has a pending entity import'])
+                    ->setStatusCode(400);
+            }
+        }
+
+        $import = $wiki->wikiEntityImports()->create([
+            'status' => WikiEntityImportStatus::Pending,
+            'started_at' => Carbon::now(),
+            'payload' => $request->all(),
```

Comment:
I'm curious about the structure of `payload` here,  do we leave it intentionally flexible?

Reply:
Right now it's mostly a debugging helper, so I thought it'd be good to make it as flexible as possible.